code/modules/SCP/SCPs/SCP-500.dm

```diff
@@ -15,13 +15,13 @@
 
 //is this shitcode? kinda. does it work? absolutely.
 //these 2 lines of code make it so you can't dissolve scp 500 in water for multiplication, do not remove it.
-/obj/item/reagent_containers/pill/scp500/afterattack(obj/target, mob/user, proximity) 
+/obj/item/reagent_containers/pill/scp500/afterattack(obj/target, mob/user, proximity)
 	return 0
 
 
 /obj/item/storage/pill_bottle/scp500
-	name = "pill bottle (Panacea)"
-	desc = "This Pill Bottle has the words panacea written in it's cover, you also notice a strange note - DUE TO UNAUTHORIZED USAGE OF MULTIPLE 500-1 INSTANCES, THIS SITE IS NOW RESTRICTED FROM HAVING OVER 10 INSTANCES OF 500-1 - O5-7."
+	name = "pill bottle (SCP-500)"
+	desc = "This Pill Bottle has \'SCP-500\' written on its label. The label also notes, \'Storage of more than 10 instances of SCP-500 are limited to secure Sites only. Contact a RAISA official for more details.\'."
```

Comment:
```suggestion
	desc = "This Pill Bottle has \'SCP-500\' written on its label. The label also notes, \'Storage of more than 10 instances of SCP-500 is limited to secure Sites only. Contact a RAISA official for more details.\'."
```

Reply:
the fact Site-53 is recognized as a non-secure site is the most hilarious thing ever

Reply:
Site-53 can be infiltrated by three raccons in a coat. We both know that.

Reply:
> Site-53 can be infiltrated by three raccoons in a coat. We both know that.

Chances are that security will do nothing about it and shoot the wrong person (it will probably be Mauricio that they will shoot).

Reply:
> > Site-53 can be infiltrated by three raccoons in a coat. We both know that.
> 
> Chances are that security will do nothing about it and shoot the wrong person (it will probably be Mauricio that they will shoot).

(as they should)

Reply:
> > > Site-53 can be infiltrated by three raccoons in a coat. We both know that.
> > 
> > 
> > Chances are that security will do nothing about it and shoot the wrong person (it will probably be Mauricio that they will shoot).
> 
> (as they should)

(correct)

Reply:
Secure Site is a term for a type of site dedicated solely to protecting high-importance objects from external threats, it’s not a name given to sites who have good security.

Reply:
> Secure Site is a term for a type of site dedicated solely to protecting high-importance objects from external threats, it’s not a name given to sites who have good security.

my point still stands, Site-53 is not safe in any way, if we had a greytide the map would be decimated in 3 seconds

Reply:
> Secure Site is a term for a type of site dedicated solely to protecting high-importance objects from external threats, it’s not a name given to sites who have good security.

Well, Site-53 fails miserably at protecting high-importance objects from external threats.

Reply:
> > Secure Site is a term for a type of site dedicated solely to protecting high-importance objects from external threats, it’s not a name given to sites who have good security.
> 
> Well, Site-53 fails miserably at protecting high-importance objects from external threats.

indeed

Damn those raccoons!

Reply:
As for my suggestion: The act of storage itself is limited to secure sites, plural sounds bad there.